### ISSUE TITLE
Placement of superhero head button

### ISSUE THREAD

**Issue by justacks** [rotblauer/openbound#16]
It took me a while to figure out that the "head figure" to click on to make a single work go into superhero mode is tiny and by my name on the upper left. 

Would be much more intuitive if up next to the version options for that document (like to the left of the delete version button). I.e. a on/off switch (something like the "for this version/for this project" option in the comment area). 

I thought clicking the head next to my name would a) do nothing, didn't know it was a button and then b) make all my works superhero since it's next to my name, not that work's title

**Comment by kstrode** [comment]
Agreed.  I actually couldn't find it but I'd only looked for a bit.  The descriptor didn't seem to match this ultimate location.

**Comment by whilei** [comment]
Bump.
